Add an overview of features for global Fleet management (#2592)
elastic/docs-content: reference/fleet/manage-elastic-agents-in-fleet.md

---
navigation_title: Manage {{agents}} in {{fleet}}
mapped_pages:
  - https://www.elastic.co/guide/en/fleet/current/manage-agents-in-fleet.html
products:
  - id: fleet
  - id: elastic-agent
---

# Centrally manage {{agents}} in {{fleet}} [manage-agents-in-fleet]

The {{fleet}} app in {{kib}} supports both {{agent}} infrastructure management and agent policy management. You can use {{fleet}} to:

- Manage {{agent}} binaries and specify settings installed on the host that determine whether the {{agent}} is enrolled in {{fleet}}, what version of the agent is running, and which agent policy is used.

> - Manage agent policies that specify agent configuration settings, which integrations are running, whether agent monitoring is turned on, input settings, and so on.

Reviewer: Wondering why you use "{{agent}}" on line 14, and "agent" on line 15.

Also, personal taste but I think "and so on" reads a bit colloquial—"more" might be slightly better for ESL folks.

Author: You are right. I was using an existing text that I rephrased a bit, but I see I could have worked on it some more. Thanks for noticing these.

Author: This wording was part of the original doc, I only changed the `*` of the unordered list to `-` (personal preference). I only fixed smaller issues I noticed in the original text, but it’s true that the doc could benefit for a more in-depth review. Anyway, I’ll fix this.

Author: I think it’s `{{agent}} binaries` because here it refers more to the binaries of Elastic Agent (the product), but the other instance should probably be just “agent”.

Advanced users who don’t want to use {{fleet}} for central management can use an external infrastructure management solution and [install {{agent}} in standalone mode](/reference/fleet/install-standalone-elastic-agent.md) instead.

::::{important}
{{fleet}} currently requires a {{kib}} user with `All` privileges on {{fleet}} and {{integrations}}. Since many Integrations assets are shared across spaces, users need the {{kib}} privileges in all spaces. Refer to [Required roles and privileges](/reference/fleet/fleet-roles-privileges.md) to learn how to create a user role with the required privileges to access {{fleet}} and {{integrations}}.
::::

To learn how to add {{agents}} to {{fleet}}, refer to [Install {{fleet}}-managed {{agents}}](/reference/fleet/install-fleet-managed-elastic-agent.md).

> To use {{fleet}} go to **Management** → **{{fleet}}** in {{kib}}. The following table describes the main management actions you can perform in {{fleet}}:

Reviewer: I believe our standard practice is to minimize exact descriptions of which menus->submenus to use for finding specific pages, since the navigation/page location can change. Something like this would be more standard:

`Find **Fleet** in the navigation menu, or use the [global search field](../../find-and-organize/find-apps-and-objects.md)`

Author: That’s good advice, thanks!

| [{{agents}}](/reference/fleet/manage-agents.md) | Enroll, unenroll, upgrade, add tags, and view {{agent}} status and logs. |
| [Policies](/reference/fleet/agent-policy.md) | Create and edit agent policies and add integrations to them. |
| [{{fleet}} enrollment tokens](/reference/fleet/fleet-enrollment-tokens.md) | Create and revoke enrollment tokens. |
| [Uninstall tokens](/solutions/security/configure-elastic-defend/prevent-elastic-agent-uninstallation.md) | ({{elastic-defend}} integration only) Access tokens to allow uninstalling {{agent}} from endpoints with Agent tamper protection enabled. |
| [Data streams](/reference/fleet/data-streams.md) | View data streams and navigate to dashboards to analyze your data. |

## Global {{fleet}} management

In {{fleet}} deployments where {{agents}} are installed in diverse locations and where data must be stored in local clusters, operators need a unified view of all agents and a central management interface for tasks like upgrades, policy organization, and metrics collection. {{fleet}} offers features to facilitate this deployment model:

> - [Remote {{es}} output](/reference/fleet/remote-elasticsearch-output.md): {{agents}} can be configured to send data to a remote {{es}} cluster while their check-in payloads are sent to the management cluster. This allows {{fleet}} in the management cluster to maintain a global view of all agents while the ingested data is routed to the agents' respective local clusters.

Reviewer: ```suggestion
- [Remote {{es}} output](/reference/fleet/remote-elasticsearch-output.md): Configure {{agents}} to send data to remote {{es}} clusters while still sending their check-in payloads to the management cluster. This allows {{fleet}} on the management cluster to maintain a global view of all agents while the ingested data is routed to the agents' respective local clusters.
```

Reviewer: Alternate suggestion: "You can configure {{agents}}..."

> - [Automatic integrations synchronization](/reference/fleet/automatic-integrations-synchronization.md) {applies_to}`stack: ga 9.1.0`: This feature allows users to install an integration once in the management cluster and to use {{fleet}} to reliably synchronize and update the integration across all remote clusters. With synchronized integrations, services like [OSquery](integration-docs://reference/osquery-intro.md) can be initiated from the management cluster, and responses from dispersed agents can be collected and displayed globally within {{fleet}}'s central management cluster.

Reviewer: ```suggestion
- [Automatic integrations synchronization](/reference/fleet/automatic-integrations-synchronization.md) {applies_to}`stack: ga 9.1.0`: Install an integration once in the management cluster and use {{fleet}} to synchronize and update the integration across all remote clusters. This enables you to initiate services like [OSquery](integration-docs://reference/osquery-intro.md) from the management cluster, and to send responses from dispersed agents to {{fleet}}'s central management cluster.
```
avoiding passive voice, simplifying
